### ISSUE TITLE
Deployment Reporting "Execution Exception (state: ValidateAzureResource) (step: Invocation) Error:   Cannot read properties of undefined (reading 'AzureWebJobsStorage')"

### ISSUE THREAD

**Issue by craigeddy** [Azure/functions-action#197]
Most recent deployment yesterday worked with no issues. No changes to our function environments or publish profile and today we have the following occurring on two Actions:

```
Successfully parsed SCM credential from publish-profile format.
Using SCM credential for authentication, GitHub Action will not perform resource validation.
(node:1100) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Error: Execution Exception (state: ValidateAzureResource) (step: Invocation)
Error:   Cannot read properties of undefined (reading 'AzureWebJobsStorage')
Error:     TypeError: Cannot read properties of undefined (reading 'AzureWebJobsStorage')
    at ResourceValidator.<anonymous> (D:\a\_actions\Azure\functions-action\v1\lib\handlers\resourceValidator.js:[17](https://github.com/craigeddy/ferretly-backend/actions/runs/5894948324/job/15989655032#step:5:18)8:40)
    at Generator.next (<anonymous>)
    at fulfilled (D:\a\_actions\Azure\functions-action\v1\lib\handlers\resourceValidator.js:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
Error: Deployment Failed!
https://github.com/Azure/functions-action/issues/181
```

yaml has:

```
      - name: 'Run Azure Functions Action'
        uses: Azure/functions-action@v1
        id: fa
        with:
          app-name: 'StagingMaintenance'
          slot-name: 'Production'
          package: '${{ env.AZURE_FUNCTIONAPP_PACKAGE_PATH }}/output'
          publish-profile: ${{ secrets.AZUREAPPSERVICE_PUBLISHPROFILE_902FD29C58B149C787FC9323230F93D7 }}
```

**Comment by cbude** [comment]
Same issue, worked yesterday and haven't changed anything but noticed a release was published 2 hours ago.

**Comment by erikgassler** [comment]
Same issue, pushes this morning were working and then pushes about an hour ago failing with the noted error.

**Comment by immanuelazn** [comment]
Also getting same issue.  Was able to deploy 14 hours ago.
Re-ran function deployment 3x, and all jobs failed

**Comment by patelchandni** [comment]
Sorry for inconvenience and thank you for reporting. I have reverted the v1 tag to 1.5.0 and actively investigating the issue with 1.5.1.

**Comment by craigeddy** [comment]
> Sorry for inconvenience and thank you for reporting. I have reverted the v1 tag to 1.5.0 and actively investigating the issue with 1.5.1.

FYI I tried a yaml that had `uses: Azure/functions-action@v1.5.0` and had the same error.

**Comment by patelchandni** [comment]
Bug fix released for v1.5.1. I'll wait for feedback on v1.5.1 before updating v1. Thanks.

**Comment by patelchandni** [comment]
v1 tag is now pointing to v1.5.1 latest release.
